package.json

```diff
@@ -114,6 +114,7 @@
 		"@yarnpkg/types": "^4.0.0",
 		"cross-env": "^7.0.3",
 		"purgecss": "^5.0.0",
-		"svgo": "^3.0.2"
+		"svgo": "^3.0.2",
+		"tunnel-rat": "^0.1.2"
```

Comment:
afaict this is unused - can we remove it?

Reply:
yep